### PR TITLE
Update/bag api

### PR DESCRIPTION
**Pull request by damm89**
Aligned kadaster app with the other contrib apps and updated it for use with the new kadaster API: https://www.kadaster.nl/zakelijk/producten/adressen-en-gebouwen/bag-api-individuele-bevragingen.

### CODE REVIEW

File: backend/src/zac/contrib/kadaster/decorators.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+from functools import wraps
+
+import requests
+from rest_framework import status
+from zds_client.client import ClientError
+
+from .exceptions import KadasterAPIException
+
+
+def catch_httperror(func: callable):
```
Comment (annashamray) [inline backend/src/zac/contrib/kadaster/decorators.py:10]: Nice decorators!

**Review by annashamray — APPROVED** [review]
Could you please add some documentation about Kadaster API, at least:
* the links to documentation and/or OAS for both BAG API and Location server 
* a couple of words how we use it
* examples of test data in the test API

it will help a lot!

**Comment by svenvandescheur** [comment]
@kelvincy wil jij de frontend reviewen?

File: frontend/zac-ui/libs/features/zaak-detail/src/lib/zaak-map/zaak-map.component.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+gu-input {
+  box-sizing: border-box;
+  position: absolute;
+  top: 5px;
+  z-index: 1000;
+  padding: 0 50px;
```
Comment (kelvincy) [inline frontend/zac-ui/libs/features/zaak-detail/src/lib/zaak-map/zaak-map.component.scss:6]: This is blocking the zoom functionality on the map

**Review by kelvincy — CHANGES_REQUESTED** [review]
Looks fine overall, just a small comment about the zoom on the chart.

**Review by kelvincy — APPROVED** [review]

**Pull request merged**